src/components/Header/Header.js

```diff
@@ -253,39 +248,9 @@ class Header extends React.Component {
     );
   }
 
-  /**
-   * handleStickyHeader()
-   * Executes Actions.updateIsHeaderSticky()
-   * with the proper boolean value to update the
-   * HeaderStore.isSticky value based on the window
-   * vertical scroll position surpassing the height
-   * of the Header DOM element.
-   */
-  handleStickyHeader() {
-    const headerHeight = this.state.headerHeight;
```

Comment:
Do we still need `this.state.headerHeight` if we remove sticky header?
So are the functions, `getHeaderHeight()` and `setHeaderHeight()`

Reply:
Yup, thanks.